lib/datasource/docker/index.ts

```diff
@@ -707,15 +707,15 @@ export class DockerDatasource extends Datasource {
       // prettier-ignore
       if (err.statusCode === 429 && isDockerHost(registryHost)) {
         logger.warn(
-          { registryHost, dockerRepository, err },
+          {registryHost, dockerRepository, err},
```

Comment:
Missing prettier run?

Reply:
If I remember correctly, `run yarn prepare:husky` should enable prettier pre-commit hook